index.js

```diff
@@ -4,23 +4,29 @@ var osTmpdir = require('os-tmpdir');
 var fs = require('graceful-fs');
 var mkdirp = require('mkdirp');
 var uuid = require('uuid');
+var PinkiePromise = require('pinkie-promise');
```

Comment:
I know, but overwriting is not a problem as `pinkie-promise` uses the native version when available. And we only override it in this scope.